src/indicators.ts

```diff
@@ -59,19 +60,26 @@ export class Indicators {
         return new Promise<Section>((resolve, reject) => {
             this.xmlParse.parseContent(xmlFile, (err, data) => {
                 if (err) { return reject(err); }
-                const section = data.find((lcovSection) => {
-                    // consider windows and linux file paths
-                    let cleanFile = file.replace(/[\\\/]/g, "");
-                    let cleanLcovFileSection = lcovSection.file.replace(/[\\\/]/g, "");
-
-                    // on Windows remove drive letter from path because of cobertura format
-                    // also convert both path to lowercase because Windows's filesystem is case insensitive
-                    if (process.platform === "win32") {
-                        cleanFile = cleanFile.substr(2).toLowerCase();
```

Comment:
based on my understanding, you are going from the bottom of the file structure then moving up so you will not encounter the drive letter issue mentioned here? https://github.com/ryanluker/vscode-coverage-gutters/pull/113

Reply:
I am not sure. I am sorry but I totally forgot that I was removing this part of code. I will add helper function for cleaning files.

Are you able to add proper test for #113 ?

Reply:
i will make a note to test this with the 2.0 release but the PR can go in without that.